standard/mock/mock_lpdb_data.lua

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
I would recommend placing this data in another folder, so not `standard`. Something like `test_assets`.